.buildkite/x-pack/pipeline.xpack.metricbeat.yml

```diff
@@ -202,6 +229,9 @@ steps:
           set -euo pipefail
           source .buildkite/scripts/install_macos_tools.sh
           cd x-pack/metricbeat && mage build unitTest
+        retry:
+          automatic:
+           - limit: 3
```

Comment:
same question as before, is there a reason we don't need retries for the two Cloud module tests below?

Reply:
Same as in previous reply - the retry policy taken from global Jenkinsfile which doesn't retry lint, cloud and k8s tests